### PR TITLE
BP-2357: Fix signers verified by

### PR DESCRIPTION
**Pull request by bauer00**

### CODE REVIEW

File: packages/verification-vue-component/src/components/Verification/items/cards/SigningCard.vue [hunk 1]
```diff
@@ -92,6 +92,20 @@
                         </li>
                     </ul>
                 </div>
+                <div v-if="singleIdentityVerifier" class="verification-entry verifier">
+                    <div class="verifier-name">
+                        <div class="label">{{ _$t('verification.result.meta.signersVerifiedBy') }}</div>
+                        <div v-if="singleIdentityVerifier.image" class="verifier-image">
+                            <!-- Workaround because old verification tool should still use the old switch logo but the redesign should use a new switch logo, needs to be removed when event structure is final -->
```
Comment (alnicoli) [inline packages/verification-vue-component/src/components/Verification/items/cards/SigningCard.vue:99]: the old tool isn't used anymore, isn't it? maybe check with fabian if we can get rid of the old logo

**Review by alnicoli — APPROVED** [review]
rest looks good

File: packages/verification-vue-component/src/components/Verification/items/cards/SigningCard.vue (hunk 1 above)
Comment (bauer00) [reply packages/verification-vue-component/src/components/Verification/items/cards/SigningCard.vue:99]: I also saw that, but that would mean that we remove the _redesign from the filename and remove the unused images. That needs some coordination, so I don't think it makes sense to do that now.

Comment (alnicoli) [reply packages/verification-vue-component/src/components/Verification/items/cards/SigningCard.vue:99]: Agree

**Pull request merged**